scripts/python/tao_build.py

```diff
@@ -692,6 +760,54 @@ def get_libstdcxx(gcc_ver):
     return None, None
 
 
+@time_stage()
+def make_package(root, args):
+    libstdcxx_path, libstdcxx_name = get_libstdcxx(args.compiler_gcc)
```

Comment:
why're we going to pack libstdc+`+.so` ?

Reply:
These are just copied from internal repo.. maybe useful for non-devtoolset build.

Reply:
Fine, let it be...